src/routes/auth.js

```diff
@@ -0,0 +1,73 @@
+const express = require('express');
+const passport = require('passport');
+const GoogleStrategy = require('passport-google-oauth2').Strategy;
+const dotenv = require('dotenv');
+import jwt  from 'jsonwebtoken';
+import model from '../database/models'
+
+dotenv.config();
+const route = express.Router();
+passport.serializeUser((user, done) => { done(null, user) })
+
+passport.deserializeUser((user, done) => { done(null, user) })
+
+passport.use(
+    new GoogleStrategy(
+        {
+            clientID: "1072588531381-nmcmlfa0m0a2tmu5lqh18bc1lrec23t9.apps.googleusercontent.com",
+            clientSecret:"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" ,
+            callbackURL:"http://localhost:3000/auth/auth/google/callback",
+            passReqToCallback: true
+        },
+        (request, accessToken, refreshToken, profile, done) => {
+          return done(null, profile)
+      }
+    )
+);
+
+route.get(
+    '/auth/google',
+    passport.authenticate('google', { scope: ['profile', 'email'] })
+);
+
+route.get(
+    '/auth/google/callback',
+    passport.authenticate('google', { failureRedirect: '/auth/login',successRedirect:'/auth/profile' }),
+    
+);
+
+route.get('/profile', successGoogleLogin);
+
+async function successGoogleLogin(req, res) {
+  if (!req.user) return res.redirect("/auth/login");
+  try {
+      const userObj = req.user._json;
+      if (userObj.email_verified != true) return res.status(401).json({ message: "Email not verified", error: true });
```

Comment:
can you change `userObj.email_verified` to `userObj.verified` in order to much with others tasks as:
<img width="841" alt="Screen Shot 2023-03-14 at 14 53 59" src="https://user-images.githubusercontent.com/112763782/225006939-2a0a9c7e-efac-4671-9de7-c83ce8dc1be2.png">